test/langtools/jdk/javadoc/lib/javadoc/tester/JavadocTester.java

```diff
@@ -528,26 +521,10 @@ public void checkOutput(String path, boolean expectedFound, String... strings) {
      * @param strings the strings to be searched for
      */
     public void checkOutput(Output output, boolean expectedFound, String... strings) {
-        checkOutput(output.toString(), outputMap.get(output), expectedFound, strings);
-    }
-
-    // NOTE: path may be the name of an Output stream as well as a file path
-    private void checkOutput(String path, String fileString, boolean expectedFound, String... strings) {
-        for (String stringToFind : strings) {
-//            log.logCheckOutput(path, expectedFound, stringToFind);
-            checking("checkOutput");
-            // Find string in file's contents
-            boolean isFound = findString(fileString, stringToFind);
```

Comment:
Since you deleted checkOutput, which was the only user of `findString(String, String)`, consider also deleting `findString(Stirng, String)`.